docs/content/operations/dashboard.md

```diff
@@ -38,10 +38,10 @@ To enable the dashboard, you need to enable Traefik's API.
     
     {!more-on-command-line.md!}
 
-??? example "Using the Configuration File"
-
-    ```toml
-    [api] 
+??? Enabling
```

Comment:
Wrapping this inside an admonition seems weird. WDYT about removing the "Enabling" admonition?